src/server/qtquick/private/wqmldynamiccreator.cpp

```diff
@@ -242,6 +242,10 @@ void WQmlCreatorComponent::create(QSharedPointer<WQmlCreatorDelegateData> data,
         if (auto item = qobject_cast<QQuickItem*>(data->object))
             item->setParentItem(qobject_cast<QQuickItem*>(parent));
         m_delegate->completeCreate();
+        if (!d->requiredProperties().empty()) {
+            d->requiredProperties().clear();
```

Comment:
Why?

Reply:
We Should check `requiredProperties` after completeCreate https://github.com/qt/qtdeclarative/blob/1b58f30087eedf42f16572d8ae1d6a5b18f3d698/src/qml/qml/qqmlcomponent.cpp#L871